src/evaluator/evaluator/evaluator.py

```diff
@@ -90,10 +90,12 @@ def __init__(self):
             ADAPTER_POINT_CLOUD_TOPIC_DICTINARY[self._adapter_name_],
         )
         self.planning_subscription = self.create_subscription(
-            PathPointArray, "planning_local", self.compute_and_publish_planning, 10
-        )
+            PathPointArray, "path_planning/path", self.planning_callback, 10)
         self.planning_gt_subscription = self.create_subscription(
-            PathPointArray, "planning_gtruth", self.planning_gt_callback, 10
+            PathPointArray, "path_planning/mock_path", self.planning_gt_callback, 10)
+
+        self.perception_subscription.registerCallback(
```

Comment:
What is this? There is already a subscription registered to the callback, this in unintended

Reply:
i think it was merging shenaningans